src/subscription_manager/gui/registergui.py

```diff
@@ -796,6 +854,31 @@ def choose_initial_screen(self):
         return False
 
 
+class FirstbootWidget(RegisterWidget):
```

Comment:
My preference would be for this to live in a firstboot specific module (if we still need it after splitting PerformUnregister into ConfirmUnregister/PerformUnregister, that is).